pod-requeue.sh

```diff
@@ -21,12 +22,31 @@ test_kubectl() {
   fi
 }
 
+
 export_pods() {
+    # Find pods in status 'OutOfcpu|InsufficientFreeCPU' without using jsonpath
+    # This is a workaround until the status.conditions maps are available in the output of a raw get
+    # kubectl get pods --no-headers=true --all-namespaces | egrep -i 'OutOfcpu|InsufficientFreeCPU'
+    # clean up files from existing run
+    rm $POD_LIST || true
+    rm $POD_DUMP_RAW_JSON || true
 
-  # Collect list of all pods matching Status conditions reason of Unschedulable OR OutOfcpu
-    kubectl get po --export --all-namespaces -o json | jq '.items[] | select(.status.conditions[].reason) | select((.status.conditions[].reason == "InsufficientFreeCPU") or (.status.conditions[].reason == "OutOfcpu"))' > $POD_DUMP_RAW_JSON
+    kubectl get po --all-namespaces | egrep -i 'OutOfcpu|InsufficientFreeCPU' > $POD_LIST
+
+    if [ ! -s $POD_LIST ]; then
+      echo "${POD_LIST} is empty. No pods found"
+    else
+      cat ${POD_LIST} | awk '/(InsufficientFreeCPU|Outofcpu)/ {print "get pod "$2 " -n "$1 " -o json"}' | xargs -L1 -t kubectl >> $POD_DUMP_RAW_JSON
```

Comment:
it's hard for me to infer what this produces. can you throw a comment with expected output in `$POD_DUMP_RAW_JSON`?